Tests/Profiler/DataCollector/BlockDataCollectorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Tests\Profiler\DataCollector;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\BlockBundle\Profiler\DataCollector\BlockDataCollector;
+use Sonata\BlockBundle\Templating\Helper\BlockHelper;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Block data collector test.
+ *
+ * @author Olivier Paradis <paradis.olivier@gmail.com>
```

Comment:
Is that because this is a copy?

Reply:
Well I copied it and then changed it to use prophecy instead of normal mocks :)